vpn/dialer.go

```diff
@@ -149,6 +167,7 @@ type device struct {
 }
 
 // Up spawns two goroutines that communicate the two halves of a device.
+// TODO(ainghazal): we probably want a way of shutting them down too => device.Close() ?
```

Comment:
Same comment as above: if you have bandwidth, please create an issue and use `TODO(issue_url)`